### ISSUE TITLE
Add note about accessibility of subscript and superscript

### ISSUE THREAD

**Issue by nigelmegitt** [w3c/imsc#621]
As discussed at https://github.com/w3c/a11y-tracking/issues/252#issuecomment-3334994063 add an informative note for implementers to be aware of the need to ensure that the presented superscript/subscript text is visible as such to assistive technology.

For example something like:

>For implementations that use the CSS vertical-align property when mapping tts:fontVariant into HTML, use of vertical-align alone is insufficient for assistive technology to identify that the glyphs concerned are superscript or subscript.

or:

>Presentation implementations should mark up their output such that assistive technology can identify that the glyphs concerned are superscript or subscript. For example, use of the CSS `vertical-align` property alone could be insufficient.
